src/TextField/TextField.js

```diff
@@ -140,9 +140,15 @@ class TextField extends Component {
      */
     floatingLabelFixed: PropTypes.bool,
     /**
+     * The style object to use to override floating label styles when focused.
+     */
+    floatingLabelFocusStyle: PropTypes.object,
+
```

Comment:
Could you remove this line cf. https://github.com/callemall/material-ui/pull/4073 😁?